### DISCUSSION TITLE
分布式系统中的Linearizable事务:时间、通信与一致性 - OpenACID Blog

### DISCUSSION THREAD

**Discussion by giscus** [openacid/openacid.github.io#41] [General]
# 分布式系统中的Linearizable事务:时间、通信与一致性 - OpenACID Blog

分布式系统中,Linearizable事务的实现需要解决事务间先后顺序的判断问题,本文深入探讨了这一难题,分析了其中的时间一致性挑战,并提出了几种解决方案和设计思路。

https://blog.openacid.com/algo/linearizable/

**Comment by lance6716** [comment]
这个 id 依赖关系的更好的设计，让我想到了 MySQL 主从同步并行复制 https://dev.mysql.com/blog-archive/improving-the-parallel-applier-with-writeset-based-dependency-tracking/
记录 last_committed 作为依赖的 ID https://dev.mysql.com/worklog/task/?id=8440
